contributing/releasing.md

```diff
@@ -13,10 +13,19 @@ Each release is driven by a single **release engineer**, who is also a Googler.
 is expected to do the following, in order of priority:
 
 - Do not break Google.
+- Cut a release every Wednesday morning before 10am EST, without exception.
 - Land a release at least once a week.
 
-If something is stopping the release engineer from achieving either of the above goals, the
-culprit code should be removed immediately from the release.
+If something is stopping the release engineer from achieving any of the above goals, the culprit
+code should be removed immediately from the release.
+
+Importantly: **do not** block the cutting of the weekly release on a PR or a piece of functionality
+you'd like to land. If it hasn't landed by 10am EST on Wednesday, it's not making it into that
+week's release. If your PR is important, cut a hotfix release in addition to the typical weekly
```

Comment:
This seems more like a reason for a Hotfix release and isn't really part of the release process itself. Do we have another section where we explain **why** we might cut a hotfix release?

Reply:
It bears emphasis because I think it may not have been clear in the past that it's more important for a release to be cut and landed than for any individual feature to make it into a given weekly release.

Reply:
Are you referring specifically to "If your PR is important, cut a hotfix release in addition to the typical weekly..." ? And your proposal being that we move that sentence to the content regarding hotfixing?

Reply:
I am referring to that sentence, yes. I think some general guidance on "when should it be a Hotfix?" would be helpful in a different part of our docs.

Reply:
I’ll add a note here linking to the hotfixing docs and move this sentence there.